lib/components/src/toolbar/toolbar.ts

```diff
@@ -0,0 +1,57 @@
+import { customElement, property } from "lit/decorators";
+import styles from './toolbar.scss';
+import { TopAppBarBase } from '@material/mwc-top-app-bar/mwc-top-app-bar-base';
+import { html } from "lit";
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'td-toolbar': CovalentToolbarBase,
+  }
+}
+
+@customElement('td-toolbar')
+export class CovalentToolbarBase extends TopAppBarBase {
+  @property({type: Number}) breadcrumbs = 0;
+  
+  static override styles = [styles];
+  override render() {
+    // Create dummy data for the toolbar breadcrumbs.
+    let path: any[] = [];
+    if(this.breadcrumbs > 5) {
+      path.push(html`<span>Section title > ... > </span>`);
```

Comment:
Wherever you have the > separator in the breadcrumbs, use this material icon instead https://fonts.google.com/icons?selected=Material+Icons&icon.query=chevron+right